frontend/src/components/SmoothingLinesConfigurator.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as uiPropTypes from '../store/uiPropTypes';
+import { getSelectedResults, getSelectedLogKeys } from '../utils';
+import Check from './FormControl/Check';
+
+const SmoothingLinesConfigurator = ({
+  project,
+  results,
+  projectConfig,
+  axisName,
+  onAxisConfigLogKeySmoothingToggle,
+}) => {
+  const { axes = {}, resultsConfig = {} } = projectConfig;
+  const axisConfig = axes[axisName] || {};
+  const { logKeysConfig = {} } = axisConfig;
+
+  const selectedResults = getSelectedResults(results, resultsConfig);
+  const selectedLogKeys = getSelectedLogKeys(logKeysConfig);
+
+  return (
+    <>
+      {selectedResults.map((resultId) => (
+        <React.Fragment key={resultId}>
+          {selectedLogKeys.map((logKey) => (
+            <Check
```

Comment:
When we have, for example, 10 results and all that 10 results have the same log key (e.g. `main/loss` ), and we toggle one checkbox, all 10 checkboxes change at the same time.
Is this behavior intended?

![Screen Shot 2019-11-06 at 22 10 56](https://user-images.githubusercontent.com/17793678/68301413-439a8e00-00e3-11ea-9108-2e62be60d2a3.png)

Reply:
I didn't intend it. I tried with only one log. Sorry. 🙇 

I adopt the same axis config instead of line setting.
In other words, the same name is grouped together, so in this example it is one `main/loss` .

Thank you for your report.